src/EditorFeatures/Core/Implementation/Suggestions/SuggestedActionsSource.cs

```diff
@@ -619,9 +637,10 @@ private static SuggestedActionSetPriority GetSuggestedActionSetPriority(CodeActi
                 }
 
                 return new SuggestedActionSet(
-                    refactoringSuggestedActions.ToImmutableAndFree(),
-                    SuggestedActionSetPriority.Low,
-                    applicableSpan);
+                    categoryName: PredefinedSuggestedActionCategoryNames.Refactoring,
+                    actions: refactoringSuggestedActions.ToImmutableAndFree(),
+                    priority: SuggestedActionSetPriority.Low,
+                    applicableToSpan: applicableSpan);
             }
 
             public async Task<bool> HasSuggestedActionsAsync(
```

Comment:
It may also be worth noting that GetSuggestedActionCategoriesAsync() supersedes HasSuggestedActionsAsync entirely and HasSuggestedActionsAsync will no longer be called.